app/helpers/user_journey_helper.rb

```diff
@@ -51,11 +51,15 @@ def certificate_expiry_count(components)
     components.map(&:current_certificates).flatten.select(&:expires_soon?).count
   end
 
+  def idp_team?
+    Team.find_by_id(params[:team_id]).team_type == TEAMS::IDP unless Team.find_by_id(params[:team_id]).nil?
+  end
+
   def idp_team_user?
     Team.find(current_user.team).team_type == TEAMS::IDP unless current_user.team.nil?
```

Comment:
I prefer  `if current_user.team.present?` since we've changed the above it more consistent

```
Team.find_by_id(current_user.team).idp? if current_user.team.present?
```
I don't think we really need two methods may just one

```
def idp_team?(team)
  Team.find_by_id(current_user.team).idp? if Team.exists?(current_user.team)
end
```
You can then call in various places

```
idp_team?(params[:team_id])
idp_team?(current_user&.team)
idp_team?(current_user&.team || param[:user_id])
```